src/containers/App.js

```diff
@@ -79,6 +77,16 @@ export class App extends Component {
       document.cookie = `lang=${e};path=/`;
     };
 
+    const oidcConfig = {
```

Comment:
The login component will also require the client_secret from the openid provider. We will need to add that config.